lib/supavisor_web/open_api_schemas.ex

```diff
@@ -150,7 +152,12 @@ defmodule SupavisorWeb.OpenApiSchemas do
             },
             users: %Schema{type: :array, items: User},
             inserted_at: %Schema{type: :string, format: :date_time, readOnly: true},
-            updated_at: %Schema{type: :string, format: :date_time, readOnly: true}
+            updated_at: %Schema{type: :string, format: :date_time, readOnly: true},
+            allow_list: %Schema{
```

Comment:
Note that it wouldn't surprise me if in the future we want to add a block list as well. No change is needed right now, if you're happy with the current schema evolving from its current state to that.

Reply:
That'd be fine. I like separate `allow_list` and `deny_list` fields I think anyways.